pinot-core/src/test/java/org/apache/pinot/core/query/reduce/StreamingReduceServiceTest.java

```diff
@@ -56,7 +56,7 @@ public void testThreadExceptionTransfer() {
               threadPoolService,
               ImmutableMap.of(routingInstance, mockedResponse),
               1000,
-              mock(BaseReduceService.ExecutionStatsAggregator.class));
+              mock(ExecutionStatsAggregator.class));
```

Comment:
could we add some test for both v1 and v2 engine for collecting and aggregating stats via different metadata blocks?